sale_order_type/models/account_move.py

```diff
@@ -19,7 +19,6 @@ class AccountMove(models.Model):
 
     @api.depends("partner_id", "company_id")
     def _compute_sale_type_id(self):
-        self.sale_type_id = self.env["sale.order.type"]
```

Comment:
```suggestion
self.sale_type_id = self.sale_type_id or self.env["sale.order.type"]
```

Reply:
Sorry, it should be record by record for avoiding singleton error, but the idea is this one.

Reply:
@pedrobaeza ready this change.